tools/preflight/preflight.go

```diff
@@ -758,7 +759,18 @@ func (o *Run) createSnapshotFromPVC(ctx context.Context, volSnapName, volSnapCla
 	o.Logger.Infof("Waiting for volume snapshot - %s created from pvc to become 'readyToUse:true'", volSnap.GetName())
 	err = waitUntilVolSnapReadyToUse(volSnap, snapshotVer, getDefaultRetryBackoffParams())
 	if err != nil {
-		return nil, err
+		if err == k8swait.ErrWaitTimeout {
+			o.Logger.Warnf("volume snapshot - %s not readyToUse (waited 300 sec). Retrying...", volSnap.GetName())
+			if err = waitUntilVolSnapReadyToUse(volSnap, snapshotVer, getDefaultRetryBackoffParams()); err != nil {
+				if err == k8swait.ErrWaitTimeout {
+					return nil, fmt.Errorf("volume snapshot - %s not readyToUse (waited 600 sec) :: %s",
+						volSnap.GetName(), err.Error())
+				}
+				return nil, err
+			}
+		} else {
+			return nil, err
+		}
```

Comment:
this is not needed.. rather update func  waitUntilVolSnapReadyToUse to use higher timeout values